mirage/config.js

```diff
@@ -100,47 +100,22 @@ export default function() {
   ///////////
 
   // POST /comments
-  this.post('/comments', (schema, request) => {
-    let requestBody = JSON.parse(request.requestBody);
-    let attributes = requestBody.data.attributes;
-    let relationships = requestBody.data.relationships;
-
+  this.post('/comments', function(schema) {
+    let attrs = this.normalizedRequestAttrs();
```

Comment:
Need to use a function callback instead of an arrow function, in order to get access to `this`. `normalizedRequestsAttributes` returns a hash containing exactly what we need to either create or update a mirage record.

Example, a comment hash will look like:

```JavaScript
{
  body: '',
  markdown: '',
  userId: ''
  projectId: ''
}
```

which is the structure we were creating manually before.